roles/custom/matrix-bridge-mautrix-signal/templates/config.yaml.j2

```diff
@@ -105,6 +109,10 @@ bridge:
     private_chat_portal_meta: default
     # Should avatars from the user's contact list be used? This is not safe on multi-user instances.
     use_contact_avatars: false
+    # Should the Signal user's phone number be included in the room topic in private chat portal rooms?
+    number_in_topic: true
+    # Avatar image for the Note to Self room.
+    note_to_self_avatar: {{ matrix_mautrix_signal_default_note_to_self_avatar_uri }}
```

Comment:
This variable had better be named `matrix_mautrix_signal_bridge_note_to_self_avatar`, constructed using:

- the `matrix_mautrix_signal_*` prefix
- the path to the variable within the config: `bridge.note_to_self_avatar`